test/CMakeLists.txt

```diff
@@ -95,9 +95,9 @@ add_executable(test_priority_thread_pool test_priority_thread_pool.cpp)
 target_link_libraries(test_priority_thread_pool PRIVATE utilities Catch2::Catch2WithMain)
 catch_discover_tests(test_priority_thread_pool)
 
-add_executable(test_tree_iteration test_tree_iteration.cpp)
-target_link_libraries(test_tree_iteration PRIVATE  search Catch2::Catch2WithMain)
-catch_discover_tests(test_tree_iteration)
+#add_executable(test_tree_iteration test_tree_iteration.cpp)
```

Comment:
What happened to the tree_iteration in general, is it not working anymore?

Reply:
Yes, see above